app/models/regional_record.rb

```diff
@@ -82,9 +81,13 @@ def self.is_record_at_date?(value, event_id, record_type, country_id, timestamp)
   }.freeze
 
   def marker
-    return 'NR' if record_scope == 'NR'
-    return CONTINENT_TO_RECORD_MARKER[continent_id] if record_scope == 'CR'
-
-    'WR'
+    case record_scope
+    when :continental
+      CONTINENT_TO_RECORD_MARKER[continent_id]
+    when :world
+      "WR"
+    when :national
+      "NR"
+    end
```

Comment:
Cool! Nitpick: The order of the case branches should match the "natural" order of the enum